README.MD

```diff
@@ -32,6 +61,9 @@ POSTGRES_PASSWORD=change_super_secret_password
 
 
 ## Creating a .desktop file for execution (ONLY WORKS ON UBUNTU OS)
+Thank can only work if and only if `docker-compose` is up and running. Thoughts about automating the `docker-compose` is to run it upon start up so that the user does not have to do anything execpt the below.
```

Comment:
```suggestion
That can only work if and only if `docker-compose` is up and running. Thoughts about automating the `docker-compose` is to run it upon start up so that the user does not have to do anything execpt the below.
```

Reply:
I also agree with the concept of automating the start up routine to "Marine proof" the application.